doc/Command-Reference.md

```diff
@@ -6787,7 +6797,7 @@ This command is used to add, delete, or replace the SNMP community.
 
 - Usage:
   ```
-  config snmp community (add | del | replace) <community> (<RO|RW> | <new_community>)
+  config snmp community add <community> <RO|RW>
```

Comment:
If `RO` and `RW` are literal strings they shouldn't be surrounded by `<>`. If the user *must* provide one of them, it should read `(RO | RW)`. If the user can optionally add one of them, it should read `[RO | RW]`.